apps/scan/backend/src/vx_interpreter.test.ts

```diff
@@ -36,15 +38,20 @@ import { Interpreter, sheetRequiresAdjudication } from './vx_interpreter';
 const interpreterOutputPath = join(__dirname, '..', 'test-output-dir/');
 emptyDirSync(interpreterOutputPath);
 
-jest.mock('@votingworks/ballot-encoder', () => {
+// mock SKIP_ELECTION_HASH_CHECK to allow us to use old ballot image fixtures
+const featureFlagMock = getFeatureFlagMock();
+jest.mock('@votingworks/utils', () => {
   return {
-    ...jest.requireActual('@votingworks/ballot-encoder'),
-    // to allow changing election definitions without changing the image fixtures
-    // TODO: generate image fixtures from election definitions more easily
-    // this election hash is for the MS demo fixture images
-    sliceElectionHash: () => 'b0260b4e9d492dab3813',
+    ...jest.requireActual('@votingworks/utils'),
+    isFeatureFlagEnabled: (flag: BooleanEnvironmentVariableName) =>
+      featureFlagMock.isEnabled(flag),
   };
 });
+beforeEach(() => {
+  featureFlagMock.enableFeatureFlag(
+    BooleanEnvironmentVariableName.SKIP_ELECTION_HASH_CHECK
+  );
+});
```

Comment:
Do we need to do any resetting after each test?

Reply:
Yeah, good call, I realized that later and have it added in an in-progress commit.